scripts/write_cmpd_to_frag_dict.py

```diff
@@ -0,0 +1,69 @@
+"""
+The goals is to create a saved dictionary mapping the compound_ID to the fragalysis structure we are using.
+i.e. `AAR-POS-0daf6b7e-1: Mpro-x1311`
+The input is the compound_tracker.csv file, the output is a yaml file (default is in data/cmpd_to_frag.yaml).
+This generates a required input for the `fauxalysis_from_docking.py` script.
+Example Usage:
+    python
```

Comment:
I believe this is missing the script name?
```suggestion
    python write_cmpd_to_frag_dict.py
```